src/app/ApplicationTemplate.Mobile/Android/AndroidManifest.xml

```diff
@@ -5,7 +5,7 @@
 		  android:versionName="1.0.0">
 
 	<uses-sdk android:minSdkVersion="26"
```

Comment:
can we update the minimum also to Android 9? sdk 28

Reply:
That's the new requirement? :)

Reply:
it should be since android 13 is out, I can open a suggestion for mobile dotnet committee